scripts/pylib/pytest-twister-harness/tests/device/data/fifo_mock.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
```

Comment:
when is this test being run?

Reply:
added to the "Twister TestSuite / Twister Unit Tests" check (workflow twister_tests.yaml) and is already passing on this PR.